Ginger/GingerCoreCommon/Repository/ApplicationModelLib/ApplicationModelBase.cs

```diff
@@ -234,6 +234,8 @@ public enum eModelParts
             Parameters
         }
 
+
+        // !!!!!!!!!!!!!!!!!!!!!!!!!
         public static List<string> GetSoapSecurityHeaderContent(ref string txtBoxBodyContent)
```

Comment:
this was added as quick solution to support Security headers, we adding the headers as string to body which in it we combine VE valus (for tokens and other)